src/builder.ts

```diff
@@ -354,16 +450,16 @@ export class SchemaBuilder {
    */
   protected finalized: boolean = false;
 
-  constructor(protected config: BuilderConfig) {
+  constructor(protected options: BuilderOptions) {
```

Comment:
Minor, but can we stay consistent w/ `Options` vs `Config`? I think everywhere should be `Config`, and if we need to change it, we could just make the config more explicit, e.g. `BuilderInternalConfig` - unless there's a reason to do some of each?

Reply:
So the underlying idea is that at the top layer is some human-centric settings that permit nullables, shorthands, etc. Once these enter through the library's entrypoint(s) though, these settings get "resolved", short hands processed, defaults supplied, etc.

To make this idea more readable in the codebase, I thought of using different terms. `options` being the higher layer, and `config` being the lower, resolved, layer.

How do you feel about that?

Another point might be that the idea was rolled out semi-spuriously, rather than uniformly across the codebase.

Reply:
I will take a different tact to reduce the diff here, that being, `config / Config` and `internalConfig / InternalConfig`.

Reply:
Done, now `options` does not show up anywhere, just `config` and variants of that.